e2e/tests/features/207-spreadsheet-import.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { expect, test } from '@playwright/test';
+
+const fileToUpload = 'e2e/tests/fixtures/test-sheet.xlsx';
+
+test('sheet file upload', async ({ page }) => {
+  await page.goto('http://localhost:4001/editor');
+  await page.getByRole('button', { name: 'Rundown menu' }).click();
+  await page.getByRole('menuitem', { name: 'Delete all events' }).click();
+
+  await page.getByRole('button', { name: 'Application settings' }).click();
+  await page.getByRole('button', { name: 'Import spreadsheet' }).click();
+
+  // workaround to upload file on hidden input
+  // https://playwright.dev/docs/api/class-filechooser
+  const fileChooserPromise = page.waitForEvent('filechooser');
+  await page.getByRole('button', { name: 'Import from spreadsheet', exact: true }).click();
+  const fileChooser = await fileChooserPromise;
+  await fileChooser.setFiles(fileToUpload);
+
+  const worksheetOption1 = await page.locator('[id="event\\ schedule"]').selectOption('Sheet2');
+  expect(worksheetOption1).toEqual(['Sheet2']);
+
+  const worksheetOption2 = await page.locator('[id="event\\ schedule"]').selectOption('test');
+  expect(worksheetOption2).toEqual(['test']);
```

Comment:
really great effort here, I wonder if we could add a `data-testId` to make this selectors simpler?